pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/GrpcSendingMailbox.java

```diff
@@ -135,17 +133,7 @@ private MailboxContent toMailboxContent(TransferableBlock block)
     long start = System.currentTimeMillis();
     try {
       DataBlock dataBlock = block.getDataBlock();
-      List<ByteBuffer> bytes = dataBlock.serialize();
```

Comment:
@gortiz : Moving this logic to `DataBlockUtils` to be able to re-use it in a future PR. Lmk if there are any concerns.

Reply:
Tagging @Jackie-Jiang to take a look as @gortiz is on leave.

Reply:
Checked with Jackie offline and he's good with merging. Will raise the next parts soon.